Firmware/language_pt.h

```diff
@@ -0,0 +1,343 @@
+/**
+ * Portuguese
+ *
+ * LCD Menu Messages
+ * Please note these are limited to 17 characters!
+ *
+ */
+
+#define(length=20) WELCOME_MSG              CUSTOM_MENDEL_NAME " pronta."
+#define MSG_SD_INSERTED                     "Cartao inserido"
+#define MSG_SD_REMOVED                      "Cartao removido"
+#define MSG_MAIN                            "Principal"
+#define MSG_DISABLE_STEPPERS                "Desativar motores"
+#define MSG_AUTO_HOME                       "Pos. de origem"
+#define MSG_SET_HOME_OFFSETS                "Def desvio origem"
+#define MSG_SET_ORIGIN                      "Definir origem"
+#define MSG_COOLDOWN                        "Arrefecer"
+#define MSG_SWITCH_PS_ON                    "Ligar alimentaçao"
+#define MSG_SWITCH_PS_OFF                   "Desligar aliment."
+#define MSG_MOVE_AXIS                       "Mover eixos"
+#define MSG_MOVE_X                          "Mover X"
+#define MSG_MOVE_Y                          "Mover Y"
+#define MSG_MOVE_Z                          "Mover Z"
+#define MSG_MOVE_E                          "Extrusora"
+#define MSG_SPEED                           "Velocidade"
+#define MSG_NOZZLE                          "Bocal"
+#define MSG_NOZZLE1                         "Bocal2"
+#define MSG_NOZZLE2                         "Bocal3"
+#define MSG_BED                             "Cama"
+#define(length=14) MSG_FAN_SPEED                       "Veloc. Ventil."
+#define MSG_FLOW                            "Fluxo"
+#define MSG_FLOW0                           "Fluxo 0"
+#define MSG_FLOW1                           "Fluxo 1"
+#define MSG_FLOW2                           "Fluxo 2"
+#define MSG_CONTROL                         "Controlo"
+#define MSG_MIN                             " \002 Min"
+#define MSG_MAX                             " \002 Max"
+#define MSG_FACTOR                          " \002 Fator"
+#define MSG_TEMPERATURE                     "Temperatura"
+#define MSG_MOTION                          "Movimento"
+#define MSG_VOLUMETRIC                      "Filamento"
+#define MSG_VOLUMETRIC_ENABLED		        "E em mm3"
+#define MSG_STORE_EPROM                     "Guardar na mem."
+#define MSG_LOAD_EPROM                      "Ler da memoria"
+#define MSG_RESTORE_FAILSAFE                "Rest. Val. Segur."
+#define MSG_REFRESH                         "\xF8" "Atualizar"
+#define MSG_WATCH                           "Ecrã inform."
```

Comment:
Is it allowed to write the 'ç' as 'c' and the 'ã' as 'a'? Or does the meaning change doing that?